go-controller/hybrid-overlay/pkg/controller/master.go

```diff
@@ -289,13 +290,87 @@ func (m *MasterController) handleOverlayPort(node *kapi.Node, annotator kube.Ann
 		}
 	}
 
+	// create a reroute on the ovn_cluster_router
+	err = createLogicalRouterPolicyForNode(node, subnets)
+	if err != nil {
+		return fmt.Errorf("failed to create logical router policy: %v", err)
+	}
+
+	return nil
+}
+
+func createLogicalRouterPolicyForNode(node *kapi.Node, subnets []*net.IPNet) error {
```

Comment:
missing cleanup?

Reply:
what cleanup is missing? the hybrid overlay port is deleted as part of `deleteOverlayPort()`